scripts/provisioning/preparecmd.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+import sys
+
+from setupcmd import SetupCmd
+
+class PrepareCmd(SetupCmd):
+  """Prepare Setup Cmd."""
+  name = "prepare"
+
+  def __init__(self, config: str):
+    """Constructor."""
+    try:
+      super(PrepareCmd, self).__init__(config)
+    except Exception as e:
+      raise e
+
+  def process(self):
+    """Main processing function."""
+    retval = 0
```

Comment:
we can just put 'pass', instead of returning any value, as no other command class is returning anything from "process"

Reply:
fixed.